tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestAttributeValues.java

```diff
@@ -261,6 +262,14 @@ public void checkElement(String element, Attributes atts) {
         }
 
         private void checkAttribute(String element, String attribute, String attrValue) {
+            if(LdmlConvertRules.isSplittableAttr(element, attribute)) {
+                final Set<String> asSet = new HashSet<>();
+                SPACE_SPLITTER.splitToList(attrValue).forEach(v -> {
+                    if(asSet.add(v) == false) {
+                        throw new IllegalArgumentException("in <" + element+ " " + attribute + "=…, value " + v + " appears twice.");
```

Comment:
This does print an error and stop the test. There may be a more graceful way to fail.